sumNumbersString.cpp

```diff
@@ -0,0 +1,35 @@
+// Given a string containing alphanumeric characters, calculate sum of all
+// numbers present in the string.
+#include <cctype>
+#include <cstdlib>
+#include <iostream>
+
+int sum_numbers_string(std::string s) {
```

Comment:
Move this file to the correct directory (probably unclassified algorithms). Also change this function name to use lower camel case.

Reply:
@arnavb done